sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/frontend/ShardingProxy.java

```diff
@@ -64,20 +85,21 @@
      */
     public void start(final int port) throws InterruptedException {
         try {
-            if (GLOBAL_REGISTRY.getShardingProperties().<Boolean>getValue(ShardingPropertiesConstant.PROXY_BACKEND_USE_NIO)) {
-                BackendNettyClientManager.getInstance().start();
-            }
             ServerBootstrap bootstrap = new ServerBootstrap();
             bossGroup = createEventLoopGroup();
+            commandExecutorService = createCommandExecutorService();
```

Comment:
if we don't use userGroup, but define the customer ExecuteService in shardingProxy, it seems not a good idea. as we have discussed in wenxin chat, sharingProxy is only netty relevant variable.

Reply:
Replace commandExecutorService with CommandExecutorContext which one likes BackendExecutorContext.